### DISCUSSION TITLE
How to output charts within a Python script

### DISCUSSION THREAD

**Discussion by Ahosseinzadeh723** [moj-analytical-services/splink#2162] [General]
The chart results in Splink are designed for Jupyter Notebook, and it seems like running it with a Python script might prevent me from getting graphical results. These chart results are important. Is there any way to obtain these results?

Examples:

linker.match_weights_chart()

linker.m_u_parameters_chart()

linker.unlinkables_chart()

**Comment by RobinL** [comment]
The returned value from those functions is an Altair chart, which can be saved with e.g. chart.save("mychart.html")  or chart.save("mychart.png")

**Reply by Ahosseinzadeh723** [reply]
Thanks Robin, I got the charts!
